### PR TITLE
feat(Core/Conf): allow to change the cost of the guild bank tabs

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
## Changes Proposed:
-  Allow guild tabs prices to be changed

## Tests Performed:
- Tested in game
image

- On the server side, the feature works fine. However, the values are hardcoded in the client. So the client will still show the default prices and won't allow you to buy a tab if you don't have the required (default) money. The server will then subtract the configured amount of money.

### 1) Make sure that the default values still work fine
- Without editing the conf, run the server
- Access the game, create a guild and check the prices of the tabs
- Try to buy tabs and verify that the default values are still correct

### 2) Make sure changing the configuration works
- Add and edit the following (new) configurations in your `worldserver.conf`:
```
Guild.BankTabCost0
Guild.BankTabCost1
Guild.BankTabCost2
Guild.BankTabCost3
Guild.BankTabCost4
Guild.BankTabCost5
```
- Restart the server
- Access the game, create a new guild and check the prices of the tabs
- Try to buy tabs and verify that the prices reflect the configuration values

## Known Issues and TODO List:

- Test default [x]
- Test new config [x] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by pklloveyou** [comment]
Tested

File: src/server/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -3301,6 +3301,23 @@ NoResetTalentsCost = 0
...
+#     Guild.BankTabCost0-5
+#        Description: Changes the price of the guild tabs.
+#        Default:     1000000  - (100 Gold)
+#                     2500000  - (250 Gold)
+#                     5000000  - (500 Gold)
+#                     10000000 - (1000 Gold)
+#                     25000000 - (2500 Gold)
+#                     50000000 - (5000 Gold)
+
+Guild.BankTabCost0 = 1000000;
```
Comment (Kitzunu) [inline src/server/worldserver/worldserver.conf.dist:3314]: Remove semic

**Comment by FrancescoBorzi** [comment]
> Tested

@pklloveyou thanks for your feedback. What have you tested exactly? Because when I tried it, only the server part worked fine:

> On the server side, the feature works fine. However, the values are hardcoded in the client. So the client will still show the default prices and won't allow you to buy a tab if you don't have the required (default) money. The server will then subtract the configured amount of money.

Despite that, I still think this feature is worth to merge.

File: src/server/worldserver/worldserver.conf.dist (hunk 1 above)
Comment (FrancescoBorzi) [reply src/server/worldserver/worldserver.conf.dist]: thanks @Kitzunu ! done

**Comment by Kitzunu** [comment]
Add info in the conf that this wont update client side?

**Review by Kitzunu — APPROVED** [review]

**Comment by pklloveyou** [comment]
Although the server needs to be restarted to take effect, it can be used normally, so it has been tested.

**Review by Helias — APPROVED** [review]

**Pull request merged**